wedding_covid_numbers.Rmd

```diff
@@ -64,40 +64,45 @@ ec_deaths
 
 Adjusting for population shows that all of the countries are roughly hanging out in the same ballpark with the exception of Germany, who either has a magic trick or is reporting deaths differently. 
 
-Interestingly, we can see that France initially followed in the footsteps of Italy. However, having started strong lockdown earlier in their trajectory, has curbed deaths and will likely end up with fewer. 
+Interestingly, we can see that France initially followed in the footsteps of Italy but has recently curbed deaths and will likely end up with fewer. This might be a result of having started strong lockdown earlier in their trajectory - as a result of Italy's early meltdown.
```

Comment:
```suggestion
Interestingly, we can see that France initially followed in the footsteps of Italy. However, having started strong lockdown earlier in their trajectory, France has curbed deaths and will likely end up with fewer. 
```

Reply:
```suggestion
Interestingly, we can see that France initially followed in the footsteps of Italy but has recently curbed deaths and will likely end up with fewer. This might be a result of having started a strong lockdown earlier in their trajectory.
```